src/Components/Layouts/SplashpageLayout/SplashpageLayout.jsx

```diff
@@ -125,6 +130,7 @@ export default function SplashpageLayout() {
         </Modal>
         {handleSwitchSplashPagePanel()}
       </div>
+      <Footer footerText={textForCopyright()} />
```

Comment:
wyt of either:
1. using children prop instead of `footerText` so it could be used like:
  ```jsx
  <Footer>
    Copyright Jess White and Michael McFaddin 2020 - {new Date().getFullYear()}
  </Footer>
  ```
2. or placing the content directly inside the <Footer/> component since this doesn't really change between different layouts